style.css

```diff
@@ -1,440 +1,818 @@
+.skills #list {
   display: flex;
+  text-decoration: none;
+  flex-direction: column;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 50px;
   align-items: center;
-  background: #f7f7f9;
-  border-radius: 8px;
-  padding-left: 14px;
+  color: #000;
+  height: 60px;
+  width: fit-content;
+  gap: 6px;
 }
 
-.skill-name {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  color: #253858;
-  padding: 24px 0 24px 16px;
-  margin: 0;
-}
+@media only screen and (min-width: 768px) {
+  .about {
+    background: #fff;
+    border-top-right-radius: 100px;
+    display: flex;
+    flex-direction: row;
+    height: fit-content;
+    padding-top: 10%;
+    padding-left: 4%;
+    padding-bottom: 10%;
+    gap: 1%;
+  }
 
-.bottom-line {
-  border-bottom: 1px solid #dfe1e6;
-}
+  .about-content1 {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+  }
 
-.formx {
-  background-color: #fff;
+  .about .about-content2 {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+  }
+
+  .skills .extra {
+    text-decoration: none;
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+  }
+
+  .extra a {
+    background: #e5e5e5;
+    text-decoration: none;
+    display: flex;
+    flex-direction: column;
+    gap: 2%;
+    height: fit-content;
+    width: 100px;
+    margin-right: 12px;
+    padding: 12px;
+  }
+
+  .skills a img {
+    padding: 2%;
+    width: 50px;
+    height: 50px;
+  }
+
+  .skills a p {
+    color: #000;
+    width: 60%;
+    justify-content: center;
+    text-decoration: none;
+    align-content: center;
+    padding-top: 14px;
+  }
 }
 
-.contact-form {
-  background-color: #6070ff;
-  border-top-left-radius: 80px;
-  padding: 114px 24px;
+.contact-me .text-me {
   display: flex;
+  gap: 20px;
   flex-direction: column;
-  gap: 12px;
-  background-image: url(./images/contact-background.png);
-  background-size: 60%;
-  background-position: top right;
-  background-repeat: no-repeat;
+  justify-items: center;
+  align-self: center;
 }
 
-.contact-title {
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  text-align: center;
-  line-height: 52px;
-  color: #fff;
-  margin: 0;
+.contact-me .text-me input {
+  font-size: 16px;
+  padding: 8px;
+  width: 290px;
+  height: 40px;
+  border-radius: 12px;
+  border-style: none;
 }
 
-.contact-text {
+.contact-me .text-me textarea {
+  font-size: 16px;
+  margin-top: 1px;
+  padding: 8px;
+  width: 290px;
+  height: 160px;
+  border-radius: 12px;
+  border-style: none;
+}
+
+.contact-me .text-me .touch {
+  color: #486493;
+  background: #fff;
   font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  text-align: center;
-  color: #ebebff;
-  margin: 0;
+  font-size: 16px;
+  width: fit-content;
+  padding: 14px;
+  border-radius: 12px;
+  border-color: #6070ff;
+}
+
+.contact-me .text-me .touch:hover {
+  color: #babfed;
+  transform: translate(0, -2px);
 }
 
 .form {
```

Comment:
- **[OPTIONAL]** Please edit uour form styles to be similar to the Figma template 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/218276809-2a73fea1-ee8b-40ca-ad27-710de97f9875.png)